examples/graph/app.js

```diff
@@ -0,0 +1,393 @@
+/* global window,document */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import {json as requestJSON, csv as requestCSV} from 'd3-request';
+
+import DeckGLOverlay from './deckgl-overlay';
+import {default as GraphBasic} from './graph-layer/adaptor/graph-basic';
```

Comment:
shall we add an index.js to the adaptor directory that handles default exports, so it's less verbose in the app.js? 
also, would be helpful to have an inline comment, like in your PR description, briefly mentioning what an adaptor does, in cases users (like me) jump directly into reading the code.